content/en/docs/languages/java/jmx.md

```diff
@@ -0,0 +1,281 @@
+---
+title: JMX Metrics
+weight: 14
+description: Collect metrics from JMX MBeans using OpenTelemetry
+cSpell:ignore: mbean mbeans jmxremote jconsole
+---
+
+This page describes how to collect metrics from
+[JMX](https://docs.oracle.com/javase/8/docs/technotes/guides/management/agent.html)
+(Java Management Extensions) MBeans and export them to OpenTelemetry.
+
+## Overview
+
+JMX (Java Management Extensions) is a Java technology that provides tools for
+managing and monitoring applications, system objects, devices, and
+service-oriented networks. JMX MBeans (Managed Beans) expose management
+attributes and operations that can be monitored.
+
+The OpenTelemetry JMX Metric Insight module allows you to bridge JMX metrics
+into OpenTelemetry, enabling you to:
+
+- Monitor JVM metrics (memory, garbage collection, threads, etc.)
+- Collect metrics from application-specific MBeans
+- Export JMX data alongside other OpenTelemetry telemetry signals
+- Use predefined metric mappings for popular frameworks
+
+## Installation
+
+### Using the Java Agent
+
+The easiest way to collect JMX metrics is using the OpenTelemetry Java Agent
+with the JMX metrics extension:
+
+1. Download the OpenTelemetry Java Agent (if not already installed):
+
+   ```sh
+   curl -L -O https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/latest/download/opentelemetry-javaagent.jar
+   ```
+
+2. Run your application with the agent and enable JMX metrics:
+
+   ```sh
+   java -javaagent:opentelemetry-javaagent.jar \
+     -Dotel.jmx.enabled=true \
+     -jar myapp.jar
+   ```
+
+### As a Standalone Library
+
+You can also use the JMX metrics library as a standalone dependency:
+
+Add the following dependency to your project:
+
+**Maven (`pom.xml`):**
+
+```xml
+<dependency>
+  <groupId>io.opentelemetry.instrumentation</groupId>
+  <artifactId>opentelemetry-jmx-metrics</artifactId>
+  <version>{{% param vers.instrumentation %}}-alpha</version>
+</dependency>
+```
+
+**Gradle (`build.gradle`):**
+
+```groovy
+dependencies {
+  implementation("io.opentelemetry.instrumentation:opentelemetry-jmx-metrics:{{% param vers.instrumentation %}}-alpha")
+}
+```
```

Comment:
even if we do decide to keep this section, let's use the gradle kotlin over groovy (we've normalized around that).